packages/components/src/navigation/level.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon, chevronLeft } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { useNavigationContext } from './context';
+import { DEFAULT_LEVEL } from './constants';
+import { BackButtonUI, MenuTitleUI, MenuUI } from './styles/navigation-styles';
+
+export default function NavigationLevel( {
+	children,
+	className,
+	level = DEFAULT_LEVEL,
```

Comment:
Not sure if this should be optional? 🤔 I guess it's a question of are we optimizing for single level use case vs multi-level. Also would `ROOT_LEVEL` be a better name for this constant?

Reply:
We are optimizing for single-level (flat), as we'll need to figure out the nesting accesibility.
Either way, making it optional for the root level means that the consumer doesn't need to remember the arbitrary name we choose to indicate the root level. 🙂 

And yes, `ROOT_LEVEL` is more clear. 👍